pkg/cmd/target/shoot.go

```diff
@@ -23,11 +23,15 @@ func NewCmdTargetShoot(f util.Factory, ioStreams util.IOStreams) *cobra.Command
 	}
 	cmd := &cobra.Command{
 		Use:   "shoot",
-		Short: "Set shoot for next operations",
-		Example: `# target shoot with name shoot_name
-gardenctl target shoot shoot_name`,
+		Short: "Target a shoot",
+		Long:  "Target a shoot to set the scope for the next operations",
+		Example: `# target shoot with name my-shoot of currently selected project
+gardenctl target shoot my-shoot
+
+# target shoot with name my-shoot of project my-project
+gardenctl target shoot my-shoot --project my-project --garden my-garden`,
```

Comment:
```suggestion
gardenctl target shoot my-shoot --garden my-garden --project my-project`,
```
better for the completion experience would be `gardenctl target shoot --garden my-garden --project my-project my-shoot` but it makes the example harder to read

Reply:
ok let's make the examples always like you would do when making use of completion functionality

Reply:
BUT keep the subcommand before the flags as you said, it is much harder to read